test/core/test_upnp_xml.cc

```diff
@@ -125,19 +125,6 @@ TEST_F(UpnpXmlTest, RenderObjectItem)
     EXPECT_STREQ(didl_lite_xml.c_str(), expectedXml.str().c_str());
 }
 
-TEST_F(UpnpXmlTest, CreatesSecCaptionInfoElement)
```

Comment:
It would be nice to add some tests too

Reply:
Added a combined test for albumart and subtitle